cdxev/set.py

```diff
@@ -16,6 +16,7 @@
 class SetConfig:
     force: bool
     allow_protected: bool
+    ignore_missing: bool
```

Comment:
why not set it to `false` per default? (must make adjustments to the default provided in the tests, then)